### ISSUE TITLE
seperate loopback server and client (ionic) application

### ISSUE THREAD

**Issue by paidon** [strongloop/loopback#405]
hello everybody,
i am working on an ionic mobile app at the moment.
i want to transition my complete backend/data persistence system to loopback and make use of all its great functionalities like offline replication and so on.
i just started with loopback but already got the api working with mongolab as a datasource and also was able to create that awesome angular js services file with the generator. however, including this in my already existing ionic app doesn't lead to offline sync, of course.
so my question is how could i include the client side of loopback into my existing ionic app, so separating the client from the server?
any ideas?

basically, in a few words:
- existing ionic/angular js app
- connect this with a new loopback backend INCLUDING offline sync

Thank You!

**Comment by denzelwamburu** [comment]
Hi @paidon  kindly check https://github.com/denzelwamburu/lb-ngIonic .A better kickstarter for your ionic + loopback app. Though offline sync hasn't been added.
